api/operator.go

```diff
@@ -304,3 +304,21 @@ func (op *Operator) LicenseGet(q *QueryOptions) (*LicenseReply, *QueryMeta, erro
 	}
 	return &reply, qm, nil
 }
+
+func (op *Operator) Metrics(q *QueryOptions) ([]byte, error) {
```

Comment:
A comment here describing what kind of []byte metrics returns might be nice

Reply:
Added comment